lib/include/event-driven/vDrawSkin.h

```diff
@@ -0,0 +1,1159 @@
+                        }
+                    }
+                    xoffset=handR.xoffset;
+                    yoffset=handR.yoffset;
+                    scaling =handR.scaling;
+                    radius = handR.radius;
+                    noise = handR.noise;
+                    max_value = handR.max_value;
+                }
+                else {
+                    yWarning("Unable to build the taxel map from file");
+                    pos = default_pos;
+                }
+            }
+
+            if(bodypart == "leftHand"){
+
+                Left_hand handL = Left_hand(-5);
+
+
+                filename1 = "skinGui/left_hand_V2_1.ini";
+                filename2 = "positions/left_hand_V2_1.txt";
+
+                rf2.setVerbose(false);
+                rf2.setDefaultContext("skinGui");            //overridden by --context parameter
+                rf2.setDefaultConfigFile(filename2); //overridden by --from parameter
+                rf2.configure(0,NULL);
+                rf2.setVerbose(true);
+
+                if (rf2.check("taxel2Repr")){
+                    check+=1;
+                    std :: cout << "Calibration data";
+                    yarp::os::Bottle calibration_data = *(rf2.find("taxel2Repr").asList());
+                    for (int i = 0; i < calibration_data.size(); i++){
+                            taxel2Repr.push_back(calibration_data.get(i).asInt());
+                    }
+                    handL.initRepresentativeTaxels(taxel2Repr);
+                }
+
+                rf1.setVerbose(false);
+                rf1.setDefaultContext("skinGui");            //overridden by --context parameter
+                rf1.setDefaultConfigFile(filename1); //overridden by --from parameter
+                rf1.configure(0,NULL);
+                rf1.setVerbose(true);
+
+                if (rf1.check("SENSORS")){
+                    check+=1;
+                    std:: cout<< "\n Sensors data"<<std::endl;
+                    yarp::os::Bottle &ini_data = rf1.findGroup("SENSORS");
+
+                    for (int i = 1; i < (ini_data.size()); i++){
+
+                        yarp::os::Bottle sensorConfig(ini_data.get(i).toString());
+
+                        //std::string type(sensorConfig.get(0).asString()); useless since forearm has always same type and hand has a precise numbering for palm and fingers.
+
+                        int id=sensorConfig.get(1).asInt();
+                        double x=sensorConfig.get(2).asDouble();
+                        double y=sensorConfig.get(3).asDouble();
+                        double orientation=sensorConfig.get(4).asDouble();
+                        double gain=sensorConfig.get(5).asDouble();
+                        int    mirror=sensorConfig.get(6).asInt();
+                        int    layoutNum=sensorConfig.get(7).asInt();
+
+                        handL.get_data(id,x,y,orientation,gain,mirror,layoutNum);
```

Comment:
`layoutNum` is not used.
```suggestion
                        handL.get_data(id,x,y,orientation,gain,mirror);
```